README.md

```diff
@@ -235,40 +241,12 @@ After the HAR file has been recorded, use [har-to-k6](https://k6.io/docs/test-au
 
 ## Authentication
 
-Authentication by eIAM through a Keycloak instance.
+Authentication is provided by the Swiss federal government's eIAM through ADFS.
 We use Next-auth to integrate our application with Keycloak.
 See https://next-auth.js.org/providers/keycloak for documentation.
```

Comment:
I think we could drop references to Keycloak altogether.

Alternatively, we could keep them (including what we had before in the Locally section below) and explain the use of Keycloak as an identity provider. Obviously more effort, but potentially more useful to other users of the project.

Reply:
Ah, I had forgotten this last mention. I removed it and pointed to the custom adfs provider. Thanks.